sklearn/model_selection/tests/test_search.py

```diff
@@ -814,6 +814,12 @@ def test_grid_search_cv_results():
     n_grid_points = 6
     params = [dict(kernel=['rbf', ], C=[1, 10], gamma=[0.1, 1]),
               dict(kernel=['poly', ], degree=[1, 2])]
+    grid_search = GridSearchCV(SVC(gamma="scale"), cv=n_splits, iid=False,
```

Comment:
Why do you need to add these lines? Seems duplicate?
https://github.com/gxyd/scikit-learn/blob/890ce677065a4ef90a5c3771e096a5876c46856b/sklearn/model_selection/tests/test_search.py#L836-L839

Reply:
Oh, I see. This isn't intentional, good catch though :) . What happened is, when I merged the master branch (2nd commit in git history), after basing my commit of @neokt (@neokt initially started working on it), I didn't resolve the commits cleanly, since there were a few lines in master that had been removed later on.

So, all in all it was a case of me not cleanly resolving merge conflicts.